src/components/card/index.ts

```diff
@@ -1,4 +1,5 @@
 export { default, default as Card } from "./card.component";
+export type { CardProps } from "./card.component";
 export { default as CardColumn } from "./card-column";
```

Comment:
should we not also export the props interfaces rom all the subsidiary components as well? (We used to provide these in `index.d.ts`.)

Reply:
Good point 👍🏼